### PR TITLE
Update Readme - contrib and install

### PR DESCRIPTION
**Pull request by Aareon**
Add pytest info
Add installation in develop-mode info

Add regular installation info

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,6 +17,14 @@
...
 }
 
+# type annotation support for Python versions 3.6.1+
+is_py36 = (version_info.major, version_info.minor) == (3, 6)
+py36_compatible = is_py36 and (version_info.micro >= 1)
+if py36_compatible:
+    requires.append("future-annotations>=1.0.0")
+    # Non-standard environments (ex. AWS Lambda, iOS_Python/Stash)
+    # See: https://github.com/asottile/future-annotations#when-you-arent-using-normal-site-registration
+
```
Comment (AnonymousDapper) [inline setup.py]: Remove this for now, let's keep the backport issue in the other PR.

File: setup.py [hunk 2]
```diff
@@ -31,6 +40,6 @@
     include_package_data=True,
     install_requires=requires,
     extras_require=requires_optional,
-    python_requires=">=3.8.0",
+    python_requires=">=3.6.1",
```
Comment (AnonymousDapper) [inline setup.py]: Same situation as future-annotations. Remove this for now and we'll keep working on it separately.

File: README.md [hunk 3]
```diff
@@ -11,5 +11,19 @@
...
+Any public interfaces should be fully-typed (this is checked by `pyright`).
+
+To run tests, execute `pytest` in the root of this repository.
+
+To install within your development environment, use either;
+`pip install -e .` or `python setup.py develop`
+
+# Installation
+Using `git-clone` and `pip`
+`git clone https://github.com/AnonymousDapper/aiotnb && cd aiotnb`
```
Comment (AnonymousDapper) [inline README.md:25]: Include a `git checkout bank-api` here, as it won't default to that branch.

File: setup.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 import re
 from pathlib import Path
+from sys import version_info
```
Comment (AnonymousDapper) [inline setup.py]: This also is part of the future-annotations code, so it goes.

**Review by AnonymousDapper — CHANGES_REQUESTED** [review]
I like the readme changes, but I want to keep the other code separate for now. Feel free to merge after addressing the other comments.

**Comment by Aareon** [comment]
Closing to fix commits. Will open a new PR.

**Pull request closed without merging**